hcipy/aperture/generic.py

```diff
@@ -515,3 +516,27 @@ def make_hexagonal_segmented_aperture(num_rings, segment_flat_to_flat, gap_size,
 		segment_positions = segment_positions.subset(mask)
 
 	return make_segmented_aperture(segment, segment_positions, return_segments=return_segments)
+
```

Comment:
Do we want to make them deprecated immediately? Or what until the next major tick?

Reply:
Never mind this comment. I read the new decorator code and I approve.

Reply:
I'm aiming to have them deprecated for at least one minor release *and* for at least a year. I unofficially do (and have done) this for all breaking changes.